src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -8,6 +8,6 @@ namespace Microsoft.Health.Fhir.SqlServer.Features.Schema
     public static class SchemaVersionConstants
     {
         public const int Max = (int)SchemaVersion.V4;
-        public const int Min = (int)SchemaVersion.V1;
+        public const int Min = (int)SchemaVersion.V4;
```

Comment:
We should include a doc in this PR that notes what would be involved in getting from 2,3 => 4. Manual steps might be fine this time, but we should document and make known this will be required

Reply:
Good point. I have updated the document to mention the manual steps needed to upgrade from 1.2 => 3. And from version 3 to 4 or higher can be upgraded via tool.